easybuild/easyconfigs/f/fastqc/fastqc-0.11.4-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'EB_Java'
```

Comment:
hmm, this is... weird

`EB_Java` is intended to install Java itself, not stuff written in Java...

@nathanhaigh: I know this is not your change, but it just stood out for me, I wonder what this is about